keps/sig-api-machinery/20180415-crds-to-ga.md

```diff
@@ -157,15 +162,24 @@ TODO: complete this list
 
 ### e2e tests for GA
 
-* Custom Resources should be readable and writable at all available versions (test for round trip-ability)
 * A Custom Resource watch should terminate if its CustomResourceDefinition is deleted or updated
 
 ### Scale Targets for GA
 
-* TODO quantify: Read/write latency of CRDs within X% of native Kubernetes types
-* TODO quantify: Latency degrades less than X% for up to 100k Custom Resources per CRD kind
-* TODO quantify: Webhook conversion QPS of a noop converter is within X% of QPS with no webhook
-* Coordinate with sig-scalability
+Scale targets here are both CRDs with and without a webhook converter.
+
+Conditions:
+
+- A trivial webhook converter with ~1ms processing latency.
+- Custom resource size is <10kb of data.
+- Kubernetes master components running on "standard" sized VM, e.g. GCE n1-standard-1.
+
+Targets:
+
+* Create latency of 200ms for CRDs  (within 50% of native Kubernetes types)
+* Create throughput of 250 writes/s (both without and with a noop webhook converter)
+* List latency of 250ms for 1k Custom Resources, 2.5 seconds for 10k Custom Resources
```

Comment:
Same here - why we're trying to do something different that for native resources?

Reply:
Fixed